hadoop-hdds/test-utils/src/main/java/org/apache/ozone/test/IntLambda.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.apache.ozone.test;
+
+import java.util.function.IntSupplier;
+
+/** Test utilities for working with lambdas returning int value. */
+public interface IntLambda {
+
+  static ToIntExecutable withTextFromSystemIn(String... lines) {
+    return runnable -> {
+      try (AutoCloseable ignored = GenericTestUtils.supplyOnSystemIn(lines)) {
+        return runnable.getAsInt();
+      } catch (RuntimeException e) {
+        throw e;
```

Comment:
We can remove this `catch` block as we are not doing anything with the caught exception other than throwing it again.

Reply:
@nandakumar131 The goal of this `catch` block is to avoid wrapping `RuntimeException` in `RuntimeException`.